tests/bluetooth/shell/audio.conf

```diff
@@ -0,0 +1,56 @@
+# Incresed stack due to settings API usage
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+CONFIG_TEST=y
+CONFIG_TEST_LOGGING_DEFAULTS=n
+CONFIG_BT=y
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_PRIVACY=y
+CONFIG_BT_SIGNING=y
+CONFIG_BT_FIXED_PASSKEY=y
+CONFIG_BT_ATT_PREPARE_COUNT=2
+CONFIG_BT_SHELL=y
+CONFIG_BT_DEVICE_NAME="test shell"
```

Comment:
Do we want "audio" in the shell name for this config?

Reply:
I think that's a good idea; added.